functionary/core/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .celery import app as celery_app
```

Comment:
I would add a comment here explaining the above so that someone reading the code understands why this is needed.  There are ways to explicitly configure where the tasks are in situations where autodiscovery won't work, but I think this solution is fine as long as we document why we did it.